schema/trade.proto

```diff
@@ -206,6 +206,16 @@ message NewOrder {
   uint64 subaccount_id = 9;
   optional SelfTradePrevention self_trade_prevention = 10;
   PostOnly post_only = 11;
+  // If true, this order will be automatically cancelled
+  // after the closure of the network connection
+  // between Cube's servers and the client that placed the order.
+  //
+  // If the client initiates the disconnect or network instability drops the connection,
+  // the order will be cancelled when Cube's servers recognize the disconnection.
+  //
+  // In the event of a server-side disconnect that causes a halt in trading,
+  // such as scheduled downtime, the order will be cancelled before trading resumes.
+  bool cancel_on_disconnect = 12;
```

Comment:
Just wonder if we want to make this flag also on Modify Order. 
Currently once the flag is set, users cannot modify this flag. They need to cancel and create a new order. 
All other fields on order seem to be modifiable.

Reply:
We sure can; considered that but left it off as non-MVP.

Would make recovery slightly harder to implement as you now have to ask the question, "was COD enabled at the time of the shutdown" rather than just looking at the NewOrder, but perhaps that's a better way to implement it anyway...